docs/using/syntax-optional.md

```diff
@@ -42,21 +212,31 @@ This text is **standard** _Markdown_
 :::
 ::::
 
-The supported directives are: admonition, attention, caution, danger, error, important, hint, note, seealso, tip and warning.
-
-These directives do **not** currently allow for parameters to be set, but you can add additional CSS classes to the admonition as comma-delimited arguments after the directive name. Also `admonition` can have a custom title.
-For example:
+and also parameter options can be used:
 
 ```md
-:::{admonition,warning} This *is* also **Markdown**
+:::{admonition} This *is* also **Markdown**
```

Comment:
just to be clear - we are now **deprecating** this original `admonition,warning` syntax, right?

Reply:
yes its deprecated. By deprecated though, I mean entered a deprecation cycle; after this PR, the code will still be parsed and create the same HTML, but a (sphinx) warning will be issued of the deprecation.
Then it will be actually removed in the version after next

This is the same for all the deprecated sphinx configuration options

Reply:
so people should still be able to make their sites/books as before, but they will get a number of warnings, if they were using these features (with messages pointing them towards how to fix)

Reply:
makes sense to me - thanks for the explanation 👍